charts/pomerium/templates/tls-secrets.yaml

```diff
@@ -187,4 +213,9 @@ data:
   {{ template "pomerium.proxy.tlsSecret.certName" . }}: {{ .Values.proxy.tls.cert | b64enc }}
   {{ template "pomerium.proxy.tlsSecret.keyName" . }}: {{ .Values.proxy.tls.key | b64enc }}
 {{-     end }}
+{{-     if and (not .Values.proxy.existingJWTSecret) .Values.proxy.jwt.signingKey }}
+---
+{{ template "pomerium.proxy.jwtSecretObject" . }}
+  {{ template "pomerium.proxy.jwtSecret.keyName" . }}: {{ .Values.proxy.jwt.signingKey | b64enc }}
```

Comment:
This keyName is named to be consistent with certName and keyName from TLS secrets, but none seems to be defined anywhere. So I suppose I'm missing something.

@travisgroth could you help me to figure out where it should be defined ?

Reply:
...oh I think you found a bug.  Those were previously defined in `_helpers.tpl` to support a legacy secret layout, which was removed in 6.0.0.  It looks like these structures weren't updated.

They are now hard coded since they're proper TLS secrets like so - https://github.com/pomerium/pomerium-helm/blob/9ea4dbcbde8533eaf44e0b4c21d5a619dba5b568/charts/pomerium/templates/tls-secrets.yaml#L166-L167

So these should be just set to the field names we settle on in the other discussion.